skyportal/handlers/api/weather.py

```diff
@@ -65,6 +72,9 @@ def get(self):
         weather_prefs = {**default_prefs, **weather_prefs}
 
         telescope_id = int(weather_prefs["telescopeID"])
+        # use the query telecope ID otherwise fall back to preferences id
+        telescope_id = self.get_query_argument("telescope_id", telescope_id)
```

Comment:
```suggestion
        default_telescope_id = int(weather_prefs["telescopeID"])
        # use the query telecope ID otherwise fall back to preferences id
        telescope_id = self.get_query_argument("telescope_id", default_telescope_id)
```

Reply:
Also, can the default telescope ID ever be undefined?